src/components/Menu/MenuGridItem/__tests__/__snapshots__/menuGridItem-snapshot-tests.jest.js.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`MenuGridItem renders correctly with a child 1`] = `
 <section
-  className="menu-grid-item--wrapper"
```

Comment:
just make sure there's no need for this classname

Reply:
I'm sure about this one